simplepath/utils.py

```diff
@@ -50,3 +51,11 @@ def deepvars(data_object):
         return object_dict
     else:
         return data_object
+
+
+def divide(a, b):
```

Comment:
so in other words, you can move this to the lookup itself and register both `/` and `//` with appropriate `operator` functions